packages/block-library/src/quote/deprecated.js

```diff
@@ -109,7 +194,11 @@ const v2 = {
 			type: 'string',
 		},
 	},
-	migrate: migrateToQuoteV2,
+	migrate( attributes ) {
+		const [ v2Attributes, innerBlocks ] = migrateToQuoteV2( attributes );
+		const v4Attributes = migrateTextAlign( v2Attributes );
+		return [ v4Attributes, innerBlocks ];
```

Comment:
This appears to be repeated quite a lot, it would make things easier to grok if we could make this more concise.